rmf_task/include/rmf_task/TaskPlanner.hpp

```diff
@@ -39,6 +39,73 @@ class TaskPlanner
 {
 public:
 
+  /// The strategy to use when assigning tasks to robots during node expansion.
+  /// This struct defines the various profiles and their associated weights
+  /// for cost calculation.
+  struct TaskAssignmentStrategy
+  {
+    enum class Profile
```

Comment:
```suggestion
    enum class Profile : std::uint32_t
```

Specifying an explicit size for the enum guarantees the ABI stability will be maintained even if we add more profile options in the future.

Reply:
address in [commit](https://github.com/kjchee/rmf_task/commit/0de02fd253726919d0e07dfd39e0baf573f496f6)